src/api.rs

```diff
@@ -232,10 +230,27 @@ impl ThreadPool {
         unsafe {
             let job_a = JobImpl::new(op, LockLatch::new());
             self.registry.inject(&[job_a.as_job_ref()]);
-            job_a.latch.wait();
+            job_a.wait();
             job_a.into_result()
         }
     }
+
+    /// Sends `op` to the work queue to be eventually executed.
+    pub fn push_and_forget<'a, OP>(&'a self, op: OP)
+        where OP: FnOnce() -> () + Send + 'static
+    {
+        unsafe {
+            // We transmute the closure to a 'static closure in order to be able
+            // to store it as a box, knowing that the job won't outlive us.
```

Comment:
I don't follow this comment. I don't see a transmute here, for example, and i'm not sure who "us" is in this sentence (it seems like the job may well outlive the current frame, for example).